prow/config/config.go

```diff
@@ -1890,15 +1903,16 @@ func setPeriodicProwJobDefaults(c *Config, ps *Periodic) {
 
 	ps.ProwJobDefault = c.mergeProwJobDefault(repo, ps.Cluster, ps.ProwJobDefault)
 }
+
 func setPresubmitDecorationDefaults(c *Config, ps *Presubmit, repo string) {
 	if shouldDecorate(&c.JobConfig, &ps.JobBase.UtilityConfig) {
-		ps.DecorationConfig = c.Plank.mergeDefaultDecorationConfig(repo, ps.Cluster, ps.DecorationConfig)
+		ps.DecorationConfig = c.Plank.mergeDefaultDecorationConfig(repo, ps.Cluster, &metav1.Duration{Duration: 10 * time.Minute}, &metav1.Duration{Duration: 48 * time.Hour}, &metav1.Duration{Duration: 5 * time.Minute}, ps.DecorationConfig)
```

Comment:
Shouldn't the user be in charge the overall defaults?

Reply:
Removed 👍